Misc/NEWS.d/next/Core_and_Builtins/2024-07-25-01-45-21.gh-issue-122239.7zh-sW.rst

```diff
@@ -0,0 +1,2 @@
+Improve the error message for unbalanced unpacking, always showing the
```

Comment:
I guess that the word "always" is unnecessary here, since the words that come right after say that this is not "always" 🙂

Reply:
Fair enough, rephrased